src/main/java/com/zepben/evolve/cim/iec61970/base/wires/PowerTransformerEnd.kt

```diff
@@ -57,10 +57,11 @@ class PowerTransformerEnd @JvmOverloads constructor(mRID: String = "") : Transfo
     var g: Double = 0.0
     var g0: Double = 0.0
     var phaseAngleClock: Int = 0
-    var r: Double = 0.0
-    var r0: Double = 0.0
+    var r: Double = Double.NaN
```

Comment:
I thought we were going to move to these becoming nullable from a public interface point of view but would store it as a Double.NaN under the covers if null or NaN was passed in, and then would return null if NaN under the covers. (And returning null would allow us to have a unified "not known" value for either floating point or integer numbers).
Your changes to the database schema / reader / writer stuff seem to imply this is the case... 

Obviously if done here, the star impedance class would need to do the same. And I guess it would make sense for the ResistanceReactance class to have nullables too then. It could probably just use Double? vals seeing it's a short lived object?